jina/drivers/search.py

```diff
@@ -31,24 +31,25 @@ def __init__(
 
 
 class KVSearchDriver(BaseSearchDriver):
-    """Fill in the doc/chunk-level top-k results using the :class:`jina.executors.indexers.meta.BinaryPbIndexer`
+    """Fill in the results using the :class:`jina.executors.indexers.meta.BinaryPbIndexer`
 
     .. warning::
-        This driver loops over all chunk/chunk's top-K results, each step fires a query.
-        This may not be very efficient, as the total number of queries depends on ``level``
+        This driver runs a query for each document.
+        This may not be very efficient, as the total number of queries grows cubic with the number of documents, chunks
+        per document and top-k.
 
-             - ``level=chunk``: D x C x K
-             - ``level=doc``: D x K
-             - ``level=all``: D x C x K
+             - ``granularity=0``: D x K
+             - ``granularity=1``: D x C x K
```

Comment:
I would reflect it with `traversal_paths` arguments and not granularity. 

If 

`traversal_paths = ['m'] => D x K`
`traversal_paths = ['r'] => D`
`traversal_paths = ['cm'] => D x C x K`
`traversal_paths = ['m', 'cm'] => D x K + D x C x K`

Is this what we want to express?

Reply:
really awesome illustration. Thanks I will take this. This is much more understandable for the user